include/btu/tex/crunch_functions.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "btu/tex/crunch_texture.hpp"
+#include "btu/tex/detail/common.hpp"
+#include "btu/tex/detail/formats_string.hpp"
+#include "btu/tex/dimension.hpp"
+#include "crunch/crn_dxt_image.h"
+#include "crunch/crn_texture_conversion.h"
```

Comment:
Please use <> includes for external libraries (and I'm still not sure of whether to use it for bethutil own libraries)

Reply:
Changed.